python/pip_install/pip_repository.bzl

```diff
@@ -713,7 +713,20 @@ def _whl_library_impl(rctx):
         )
 
     result = rctx.execute(
-        args + ["--whl-file", whl_path],
+        args + [
+            "--whl-file",
+            whl_path,
+            # TODO @aignas 2023-12-04: this is not the best way right now
+            # so if this approach is generally good, this can be refactored, by
+            # using the names of the config_setting labels.
+            "--platform=linux_x86_64",
```

Comment:
Added extra platforms and moved the definition to an `attr` on the `repository_rule`. It may still need us updating the `wheel.py` file for any unseen platforms, but I made it a little bit more robust.